test/stage1/behavior/vector.zig

```diff
@@ -632,3 +632,17 @@ test "arrays - indexed SIMD store" {
     // Requires quite a bit of work
     //comptime S.doTheTest();
 }
+
+test "bounds checks - loads" {
+    const S = struct {
+        fn doTheTest() void {
+            var v = [_]u32{7, 8};
+            var s: @Vector(2, usize) = [_]usize{1, 0};
+            var res = v[s];
+            expect(res[0] == 8);
+            expect(res[1] == 7);
+        }
+    };
+    S.doTheTest();
+    // comptime S.doTheTest();
```

Comment:
Perhaps I should just remove this, as bounds checks only make sense for run-time code.